src/utils/dispute-utils.js

```diff
@@ -306,6 +334,15 @@ export function getAdjudicationPhase(dispute, round, now, courtConfig) {
   }
 }
 
+/**
+ * @dev Terminology here will be:
+ *            Last round => dispute's last round
```

Comment:
```suggestion
 *            Last round => last round actually reached in a dispute
```

This might be clearer